restaurant/views/login_view.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.auth import authenticate, login
+from django.http import HttpResponse, HttpResponseRedirect
+from django.shortcuts import render
+
+
+def login_view(request):
+"""Checks whether user exists in database and has permissions to login.
+"""
```

Comment:
Please, refactor this according to PEP 257.
> The closing quotes are on the same line as the opening quotes. This looks better for one-liners.
[https://www.python.org/dev/peps/pep-0257/#one-line-docstrings](url)